quipucords/scanner/network/normalizer.py

```diff
@@ -0,0 +1,76 @@
+"""normalizer for network scans."""
+
+import ipaddress
+
+from constants import DataSources
+from fingerprinter import formatters
+from scanner.normalizer import BaseNormalizer, FactMapper
+
+
+def infrastructure_type_normalizer(*, virt_what_type, virt_type):
+    """Normalize infrastructure type."""
+    # https://github.com/quipucords/quipucords/blob/e65d0e3deb23adca41ceed701e4d002187e593f6/quipucords/api/common/entities.py#L92-L112
+    if virt_what_type == "bare metal":
+        return "physical"
+    elif virt_type or virt_what_type:
+        return "virtualized"
+    return "unknown"
```

Comment:
I think this implementation is wrong (and will need to be updated after current @infinitewarp work on this fact), so don't need to consider the logic here. The important thing is the "demo" for a normalizer with multiple raw facts dependencies.